tests/sample/bindmonitor.c

```diff
@@ -16,15 +16,15 @@ typedef struct _process_entry
 } process_entry_t;
 
 #pragma clang section data = "maps"
-ebpf_map_definition_t process_map = {
-    .size = sizeof(ebpf_map_definition_t),
+ebpf_map_definition_in_file_t process_map = {
```

Comment:
As we have changed the name of this struct, will this cause problems with portability of ebpf programs?